.github/actions/update-apt-packages/update-apt-packages.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+UPDATED_PACKAGES_JSON="[]"
```

Comment:
shouldn't this be an empty object instead of an empty list? `"[]"` vs `"{}"`?

Reply:
No, it is really meant to be a list.